tox.ini

```diff
@@ -85,16 +87,16 @@ commands =
 [testenv:coverage-report]
 description = Create test coverage report
 deps =
-    pytest
     coverage[toml]
+    pytest
     -r{toxinidir}/requirements.txt
 commands =
     coverage report
 
 [testenv:test-build]
 description = build oci image for test and push to the local microk8s registry
 commands =
-    docker build -t wordpress:test .
+    docker build -t wordpress:test -f wordpress.Dockerfile .
```

Comment:
Why building the image here? The pipeline already does that

Reply:
This is being used to as a helper shortcut to facilitate development. I think it would be nice to have the process of local development documented since it's not as straight forward(openrc, kube-config, custom built dockerfile, microk8s registry).
It is being used in testing upgrade from older podspec charm to sidecar charm.
Since documenting the development guide(DEVELOPMENT.md) is out of this scope, would it be fine if I raise an issue for now and remove this piece of helper code? I'd be happy to write the documentation for development for it :)